packages/node/src/indexer/indexer.manager.ts

```diff
@@ -49,8 +50,13 @@ export class IndexerManager {
     });
     const tx = await this.sequelize.transaction();
     this.storeService.setTransaction(tx);
+
+    const inject = block.specVersion !== this.prevSpecVersion;
+    this.prevSpecVersion = block.specVersion;
```

Comment:
the change of this.prevSpecVersion should be move into try block, after setBlockhash, in case setBlockhash fails

Reply:
reordered the code to prevent the exception happen